src/extension.ts

```diff
@@ -1,52 +1,134 @@
-import * as vscode from 'vscode';
-import { GameProvider } from './game-provider';
-import metaData, { MetaData } from './game/meta-data';
-import getHtmlContent from './util/get-html-content';
-import createIntroPanel from './util/create-intro-panel';
-import createWebviewPanel from './util/create-webview-panel';
-
-//WebView 관리 맵(game.id, panel instance)
+import * as vscode from "vscode";
+import { GameProvider } from "./game-provider";
+import { ScoreBoardProvider } from "./score-board-provider";
+import metaData, { MetaData } from "./game/meta-data";
+import getHtmlContent from "./util/get-html-content";
+import path from "path";
+import fs from "fs";
+
+//WebView manage map(game.id, panel instance)
 const openWebviews = new Map<string, vscode.WebviewPanel>();
 
+// Function for creating directories
+function ensureDirectoryExistence(directoryPath: string){
+  if (!fs.existsSync(directoryPath)) {
+    fs.mkdirSync(directoryPath, { recursive: true});
+  }
+}
+
+// Game score generation function for local scoreboard(Use globalStorage)
+function saveScore(context: vscode.ExtensionContext, gameId: string, scoreData: { player: string, score: number }) {
```

Comment:
Move this function to util!
Make sure it should be a pure functions(for testing and lesser side-effects)